sdk/README.md

```diff
@@ -18,33 +17,67 @@ a basic sequential pipeline.
 
 ## Steps
 
-1. Setup Python environment with Conda or a Python virtual environment:
+1. Clone the kfp-tekton repo:
+    - `git clone https://github.com/kubeflow/kfp-tekton.git`
+    - `cd kfp-tekton`
+
+
+2. Setup Python environment with Conda or a Python virtual environment:
 
     - `python3 -m venv .venv`
     - `source .venv/bin/activate`
 
-2. Build the compiler:
+3. Build the compiler:
 
     - `pip install -e sdk/python`
 
-3. Run the compiler tests (optional):
+4. Run the compiler tests (optional):
 
     - `./sdk/python/tests/run_tests.sh`
 
-4. Compile the sample pipeline:
+5. Compile the sample pipeline:
 
     - `cd sdk/samples`  
     - `dsl-compile-tekton --py parallel_join.py --output pipeline.yaml`
     
-5. Run the sample pipeline on a Tekton cluster:
+6. Run the sample pipeline on a Tekton cluster:
 
     - `kubectl apply -f pipeline.yaml`
     - `tkn pipeline start parallel-pipeline`
 
+   You should see messages asking for default URLs like below. Press `enter` and take the defaults
+    ```bash
+      ? Value for param `url1` of type `string`? (Default is `gs://ml-pipeline-playgro 
+      ? Value for param `url1` of type `string`? (Default  is `gs://ml-pipeline-playground/shakespeare1.txt`) gs://ml-pipeline-
+      playground/shakespeare1.txt
+      ? Value for param `url2` of type `string`? (Default is `gs://ml-pipeline-playgro? Value for param `url2` of type `string`? (Default 
+      is  `gs://ml-pipeline-playground/shakespeare2.txt`) gs://ml-pipeline-playground/shakespeare2.txt
+ 
+      Pipelinerun started: parallel-pipeline-run-th4x6
+      
+      In order to track the pipelinerun progress run:
+      tkn pipelinerun logs parallel-pipeline-run-th4x6 -f -n default
+    ```
+ 7.  Run the command output in previous step to track the logs of the running Tekton Pipeline.
+
+      ```bash
+      tkn pipelinerun logs parallel-pipeline-run-th4x6 -f -n default
+      ```
+   
+      You should see an output similar to the one below
+      
+      ```bash
+      [gcs-download-2 : gcs-download-2] I find thou art no less than fame hath bruited And more than may be gatherd by thy shape Let my    
+      presumption not provoke thy wrath
+      [gcs-download : gcs-download] With which he yoketh your rebellious necks Razeth your cities and subverts your towns And in a moment         makes them desolate
+      [echo : echo] Text 1: With which he yoketh your rebellious necks Razeth your cities and subverts your towns And in a moment makes           them desolate
+      [echo : echo] Text 2: I find thou art no less than fame hath bruited And more than may be gatherd by thy shape Let my presumption not 
+      provoke thy wrath
+      ```
 
 ## Tested Versions
 
  - Python: `3.7.5`
  - Kubeflow Pipelines: [`0.2.2`](https://github.com/kubeflow/pipelines/releases/tag/0.2.2)
  - Tekton: [`0.11.0`](https://github.com/tektoncd/pipeline/releases/tag/v0.11.0-rc1)
- 
\ No newline at end of file
+ 
```

Comment:
since we rely on the `tkn` CLI here, and their behavior changed quite a bit in just 2 minor releases since Dec 2019, we might want to add another line for the `tkn` executable version:
```
- `tkn` CLI: [`0.8.0`](https://github.com/tektoncd/cli/releases/tag/v0.8.0)
```